testcontainers/src/core/env/config.rs

```diff
@@ -150,6 +199,15 @@ impl Config {
     }
 }
 
+/// Checks if the path exists and returns it if it does.
+fn check_path_exists(path: String) -> Option<String> {
```

Comment:
couple of alternative names:
- `get_existing_path`
- `filter_existing_path`
- `validate_path`


`check` usually doesn't imply "return/wrap" semantic

Reply:
Done!